tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -283,7 +283,7 @@ def parameters(cls):
             'threshold': NumberField(value_type=float, optional=True, min_value=0, default=0.001,
                                      description="Minimal objectiveness score value for valid detections."),
             'outputs': ListField(
-                optional=True, default=[],
+                optional=False,
```

Comment:
By default `optional=False`, I think you should remove it.

Reply:
fixed